packages/core/src/command/consolidated/consolidated-filter-by-resource.ts

```diff
@@ -0,0 +1,25 @@
+import { Bundle } from "@medplum/fhirtypes";
+import { ResourceTypeForConsolidation } from "@metriport/shared/medical";
+
+/**
+ * Filters resources in a FHIR Bundle by date.
+ *
+ * @param bundle The FHIR Bundle to filter.
+ * @param dateFrom The start date for filtering (inclusive).
+ * @param dateTo The end date for filtering (inclusive).
```

Comment:
dateFrom and dateTo in jsdoc arent in function

Reply:
good catch